src/theme/components/card.ts

```diff
@@ -0,0 +1,23 @@
+import { cardTheme as chakraCardTheme } from "@chakra-ui/theme/components/card";
+import { cardAnatomy } from "@chakra-ui/anatomy";
+import { createMultiStyleConfigHelpers } from "@chakra-ui/react";
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(cardAnatomy.keys);
+
+const variants = {
+  ...chakraCardTheme?.variants,
+  calm: definePartsStyle({
+    ...chakraCardTheme?.variants?.unstyled,
+    container: {
+      padding: "0.75rem",
```

Comment:
You can use the space tokens for padding, so this can be `3` instead of `"0.75rem"`.